app/views/sessions/_filters.html.erb

```diff
@@ -3,28 +3,87 @@
+              "bg-red text-white": params[:starts_at] == session_date.to_s,
+              "bg-white text-red": params[:starts_at] != session_date.to_s
+            ]
+          ) %>
     <% end %>
   </div>
 
-  <div class="flex items-center justify-center ml-3 p-2 bg-white rounded-full min-w-[40px] min-h-[40px]">
-    <%= inline_svg_tag("icons/filters.svg", size: "18") %>
+  <div class="flex items-center justify-center ml-3 p-2 rounded-full min-w-[40px] min-h-[40px] <%= active_session_filters? ? "bg-red text-white" : "bg-white text-red" %>">
+    <div class="relative" data-controller="dropdown">
+      <button type="button" data-action="dropdown#toggle click@window->dropdown#hide" class="flex items-center justify-center p-2 rounded-full">
+        <%= inline_svg_tag("icons/filters.svg", size: "18") %>
+      </button>
+
+      <div
+        data-dropdown-target="menu"
+        class="hidden transition transform origin-top-right absolute right-0"
+        data-transition-enter-from="opacity-0 scale-95"
+        data-transition-enter-to="opacity-100 scale-100"
+        data-transition-leave-from="opacity-100 scale-100"
+        data-transition-leave-to="opacity-0 scale-95">
+        <div class="pt-2 pb-2 bg-white rounded-lg shadow-[0_10px_50px_0_rgba(0,0,0,0.40)] flex-col inline-flex absolute top-4 right-[-8px] right-0 w-40 max-w-40">
+          <%= form_with url: sessions_path, class: "grid", method: :get do |form| %>
+            <%= form.hidden_field :starts_at, value: params[:starts_at] %>
+
+            <% SessionQuery::STATUS_SCOPES.each do |name| %>
+              <div class="self-stretch px-4 py-2 items-center gap-2 inline-flex">
+                <%= form.check_box name, checked: params[name] == "1", class: "w-4 h-4 rounded text-red", skip_base_styling: true, include_hidden: false %>
+                <div class="text-gray-800">
+                  <%= form.label name, class: "text-base font-medium" do %>
+                    <%= t("views.status_filters.#{name}") %>
+                  <% end %>
+                </div>
+              </div>
+            <% end %>
+            <div class="h-6 px-2.5 py-4 rounded justify-center items-center gap-2.5 inline-flex">
+              <div class="text-lg text-red italic underline font-black">
+                <%= form.submit t("views.status_filters.submit"), name: "", class: "text-lg text-red italic underline font-black" %>
+              </div>
+            </div>
+          <% end %>
+        </div>
+      </div>
+    </div>
   </div>
 </div>
+
+<% if active_session_filters? %>
+  <div class="flex w-full mb-2 gap-2">
+    <div class="flex flex-wrap ">
+      <div class="h-8 py-1 text-base font-bold leading-tight">Active filters:</div>
+    </div>
+    <div class="flex flex-wrap gap-2">
+      <% SessionQuery::STATUS_SCOPES.each do |name| %>
+        <% next unless params[name] == "1" %>
+        <div class="h-8 px-4 py-2 bg-green-500 bg-opacity-[.08] rounded-lg gap-2 inline-flex">
```

Comment:
The style of each filter should be different
<img width="538" alt="Screenshot 2024-08-15 at 3 23 58 p m" src="https://github.com/user-attachments/assets/fd4b35e2-99e6-4584-988f-92eaf044a9e2">